test/factories/benchmark.rb

```diff
@@ -5,7 +5,7 @@
     title { Faker::Lorem.sentence }
     description { Faker::Lorem.paragraph }
     ref_id { "xccdf_org.ssgproject.content_benchmark_RHEL-#{os_major_version}" }
-    sequence(:version) { |n| "0.#{(n / 50).floor}.#{n % 50}" }
+    sequence(:version) { |n| "100.#{(n / 50).floor}.#{n % 50}" }
```

Comment:
Our benchmarks are in the format of `0.y.z` which sometimes matched a random generated here.